solidity/contracts/ConnextVestingWallet.sol

```diff
@@ -0,0 +1,67 @@
+// SPDX-License-Identifier: MIT
+pragma solidity 0.8.20;
+
+import {VestingWalletWithCliff} from './VestingWalletWithCliff.sol';
+import {IERC20} from '@openzeppelin/contracts/token/ERC20/IERC20.sol';
+
+contract ConnextVestingWallet is VestingWalletWithCliff {
+  address constant BENEFICIARY_ADDRESS = address(420);
+  address constant CONNEXT_TOKEN_ADDRESS = address(420);
+
+  uint64 constant ONE_YEAR = 365 days;
+  uint64 constant AUG_01_2023 = 12_345_678;
+  uint64 constant VESTING_OFFSET = ONE_YEAR - ONE_YEAR / 12;
+
+  uint64 constant VESTING_START_DATE = AUG_01_2023 + VESTING_OFFSET;
+  uint64 constant VESTING_DURATION = ONE_YEAR * 13 / 12;
```

Comment:
imo it should be just 1 year

Reply:
TOTAL_AMOUNT / ONE_YEAR will give you 1/12 every month